### ISSUE TITLE
Domínio idvogados.com.br como hub para o projeto

### ISSUE THREAD

**Issue by guilfer-dev** [thr0w/idvogados#1]
Seria interessante se utilizássemos o domínio já registrado para podermos estruturar melhor o projeto, fazer um "marketing" do mesmo e quando numa etapa mais avançada agaranhar recursos (doações) para colocar o projeto no ar e mante-lo funcionando..

**Comment by teintinu** [comment]
Sim, precisamos. Mas acho que a prioridade agora é implementar as funcionalidades básicas, em especial sobre como será a hospedagem. 

De qualquer forma que aguardar a Clara Cardoso CLCAR490 nos ceder o contato técnico.

**Comment by Jairo-2809** [comment]
Olá, eu sou UX Writer. Posso colaborar com os conteúdos da plataforma. Revisões e melhorias para garantir o melhor design instrucional.

**Comment by vanderlpp** [comment]
Ainda precisando de hospedagem??? 

Que tipo?? Quais os requisitos da mesma??
